_episodes/02-meet-docker.md

```diff
@@ -58,7 +58,7 @@ $ docker container ls
 CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
 ~~~
 {: .output}
-(The command `docker info` will achieve a similar end but produces a larger amount of output.)
+(The command `docker system info` will achieve a similar end but produces a larger amount of output.)
```

Comment:
I misread this initially and interpreted the text "_The command `docker system info` will achieve a similar end..._" to meant that it would do the same as `docker container ls` which is of course not the case. I wonder if a slight rewording here could be useful, e.g.:

```suggestion
(The command `docker system info` could also be used to verify that Docker is correctly installed and operational but it produces a larger amount of output.)
```